### ISSUE TITLE
Location

### ISSUE THREAD

**Issue by Obadha** [Obadha/baraza#7]
The application will depend heavily on location.

Which path should be taken? Provide user with field to fill in their location, or have the application auto-detect user's location?

**Comment by valentineMunikah** [comment]
Providing the user with an option to select their location is much better. For the case of having the app to detect the user's location will have some draw backs as some people don't like activating location on their mobile phones due to security purposes.

**Comment by jvallera** [comment]
I agree. I think it is best for people to have the option to add a location.
